context/app/static/js/components/Detail/Summary/Summary.jsx

```diff
@@ -27,11 +27,15 @@ function Summary(props) {
     description,
     status,
     children,
+    collectionName,
   } = props;
 
+  const createdDate = create_timestamp ? new Date(create_timestamp).toDateString() : 'Undefined';
+  const modifiedDate = last_modified_timestamp ? new Date(last_modified_timestamp).toDateString() : 'Undefined';
```

Comment:
The [portal index](https://github.com/hubmapconsortium/search-api/blob/master/src/elasticsearch/addl_index_transformations/portal/translate.py#L85-L95) has these in a human readable form, so just use `mapped_create_timestamp` and `mapped_last_modified_timestamp`.

Reply:
`Summary` is being used for `Collection` as well which won't have those fields since we're not getting is data from elasticsearch. Is that correct?

Reply:
That works - Let's keep this on the radar so when Collections are in ES we can simplify.